rector.php

```diff
@@ -154,4 +154,8 @@
     $services->set(FuncGetArgsToVariadicParamRector::class);
     $services->set(MakeInheritedMethodVisibilitySameAsParentRector::class);
     $services->set(SimplifyEmptyArrayCheckRector::class);
+    $services->set(TypedPropertyRector::class)
+        ->configure([
+            TypedPropertyRector::PRIVATE_PROPERTY_ONLY => true,
```

Comment:
private property only option will be removed in next rector release https://github.com/rectorphp/rector-src/pull/1496, to have default change:

- change private property
- change protected property for final class.
- don't update public property.

We can retry re-run when on next rector release.